packages/plugins/rubygem-foreman_google/rubygem-foreman_google.spec

```diff
@@ -0,0 +1,107 @@
+# template: foreman_plugin
+
+%global gem_name foreman_google
+%global plugin_name google
+%global foreman_min_version 3.5.0
+
+Name: rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?foremandist}%{?dist}
+Summary: Google Compute Engine plugin for the Foreman
+License: GPLv3
+URL: https://github.com/theforeman/foreman_google
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: ruby >= 2.5
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.5
+BuildRequires: ruby >= 2.5
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(google-apis-compute_v1) >= 0.14
+BuildRequires: rubygem(google-apis-compute_v1) < 1
+BuildRequires: rubygem(google-cloud-compute) >= 0.2
+BuildRequires: rubygem(google-cloud-compute) < 1
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+BuildRequires: npm(@babel/core) >= 7.7.0
+BuildRequires: npm(@babel/core) < 8.0.0
+BuildRequires: npm(@theforeman/builder) >= 10.0.0
+BuildRequires: npm(@theforeman/builder) < 11.0.0
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+BuildRequires: npm(react-intl) >= 2.8.0
+BuildRequires: npm(react-intl) < 3.0.0
+# end package.json dependencies BuildRequires
+
+%description
+Google Compute Engine plugin for the Foreman.
+
+
+%package doc
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -n  %{gem_name}-%{version}
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build ../%{gem_name}-%{version}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%foreman_bundlerd_file
+%foreman_precompile_plugin -s
```

Comment:
You mentioned there is no React UI, is there javascript at the ERB layer? If not, this step should not be needed.